pkgs/tools/nix/bootspec/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+rustPlatform.buildRustPackage {
+  pname = "bootspec";
+  version = "2022-05-31-unstable";
```

Comment:
```suggestion
  version = "unstable-2022-05-31";
```

Reply:
Don't we do `date-unstable` so that it can be version-sorted with everything else? Or am I misremembering?